app/javascript/bundles/RestaurantsApp/components/RestaurantsApp.jsx

```diff
@@ -266,14 +367,35 @@ export default class RestaurantsApp extends React.Component {
                         label={"Only kosher"}
                         inputName={"isKosher"}
                     />
-                </div>
-                <div className="restaurants-wrapper">
-                    <Restaurants
-                        restaurants={this.state.displayedRestaurants}
-                        onRestaurantsChange={this.onRestaurantsChange}
+                    <img
+                        className={clearBtnClass}
+                        onClick={this.clearFilters}
```

Comment:
I, personally, don't like `onClick`s on non-button or link elements...
But that's just me.